src/main/webapp/assets/modules/app.js

```diff
@@ -154,14 +154,14 @@ techlooper.config(["$routeProvider", "$translateProvider", "$authProvider", "loc
           if (window.location.host.indexOf("hiring") >= 0) {
             return "/home";
           }
-          return "/pie-chart";
+          return "/home";
         }
       });
   }]);
 
 techlooper.run(function (shortcutFactory, connectionFactory, loadingBoxFactory, cleanupFactory,
                          tourService, signInService, historyFactory, userService, routerService, $location,
-                         utils, $rootScope, $translate, jsonValue, $location) {
+                         utils, $rootScope, $translate, jsonValue, $location, $anchorScroll) {
```

Comment:
Line is too long.<br>'$location' is already defined.<br>'$anchorScroll' is defined but never used.